### PR TITLE
AI-89: Fix contact record change log issues

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
**Before**

log-issue

**After**

resolved-log-issue

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_change-log.scss [hunk 1]
```diff
@@ -1,22 +1,60 @@
...
+  .form-item {
+    border-radius: $border-radius-base;
+    box-shadow: $crm-form-item-shadow;
+    padding: 0;
   }
 
   table {
+    border-radius: $border-radius-base;
+    box-shadow: none !important;
     @include mx-expandable-table();
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_change-log.scss]: Plz add @include before the properties

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_change-log.scss [hunk 2]
```diff
@@ -1,22 +1,60 @@
...
 
     .columnheader {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_change-log.scss]: Can you avoid some hierarchy here by doing  .columnheader th {

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_change-log.scss (hunk 1 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_change-log.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_change-log.scss (hunk 2 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_change-log.scss]: Fixed.

**Pull request merged**